### PR TITLE
Fixed readme

### PR DESCRIPTION
**Pull request by jessica-tw**
Signed-off-by: jessica-tw <jessica.olinda@zup.com.br>

- Fixed docs readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,27 @@
-# CharlesCD Documentation
+# **CharlesCD Documentation**
 
-This project contains all documentation of **[CharlesCD](https://charlescd.io/)**.
+his repository uses [**hugo server**](https://gohugo.io/commands/hugo_server/) and contains **[CharlesCD](https://charlescd.io/)** official documentation.
```
Comment (barbararochazup) [inline README.md:3]: Missing **T** letter on This

File: README.md [hunk 2]
```diff
@@ -19,54 +30,73 @@ To edit, run and build this project, you will need:
...
+## **How to contribute?**
+
+You can suggest a change, a fix, or improvements to our documentation, such as: 
 - Add missing information.
 - Fix a grammar or code error.
 - Suggest a new section.
 - Translate. 
+
+###  **Contributing Guide**
+Check out Ritchie's [**contributing guide**](https://github.com/ZupIT/charlescd/blob/main/CONTRIBUTING.md).
```
Comment (barbararochazup) [inline README.md:76]: Change Ritchie's to CharlesCD 's

Another question, we have a CONTRIBUTIND.md on docs repo https://github.com/ZupIT/docs-charles/blob/main/CONTRIBUTING.md.  Why reference to charlescd repo?

File: README.md [hunk 3]
```diff
@@ -19,54 +30,73 @@ To edit, run and build this project, you will need:
...
 - Suggest a new section.
 - Translate. 
+
+###  **Contributing Guide**
+Check out Ritchie's [**contributing guide**](https://github.com/ZupIT/charlescd/blob/main/CONTRIBUTING.md). 
+
 ### **Open an issue**
 Follow the steps below: 
+
 **Step 1.** Access [**GitHub Issues**](https://github.com/ZupIT/docs-horusec/issues) and open an issue;
```
Comment (barbararochazup) [inline README.md:81]: Change link to https://github.com/ZupIT/docs-charles/issues

**Review by barbararochazup — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jessica-tw) [reply README.md:76]: Adding this one ;)

**Review by barbararochazup — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,28 +1,94 @@
...
+- Is this the first CharlesCD repository you visit? Check out other [**repositories**](https://github.com/ZupIT/charlescd).
+
+Every time that a new release is created, we save the current work into a new branch, called `release-VERSION`.
+
+## **Code of Conduct**
+Please read the [**Code of Conduct**](https://github.com/ZupIT/docs-charles/blob/main/CODE_OF_CONDUCT.md).
+
+## **Community**
+
+If you have any questions or ideas, let's chat in our [**Zup Open Source Forum**](https://forum.zup.com.br).
```
Comment (mdcpmoreira) [inline CONTRIBUTING.md:94]:
```suggestion
Do you have any question about CharlesCD? Let's chat in our [**Zup Open Source Forum**](https://forum.zup.com.br).
```

File: README.md [hunk 5]
```diff
@@ -19,54 +30,50 @@ To edit, run and build this project, you will need:
...
+Please read the [**Code of Conduct**](https://github.com/ZupIT/docs-charles/blob/main/CODE_OF_CONDUCT.md).
 
-Every time that a new release is created, we save the current work into a new branch, called `release-VERSION`,
+## **Community**
 
-## Community
-Feel free to reach out to us at:
-- [**GitHub Issues**](https://github.com/ZupIT/docs-charles/issues).
-- [**Zup Open Source Forum**](https://forum.zup.com.br).
+If you have any questions or ideas, let's chat in our [**Zup Open Source Forum**](https://forum.zup.com.br) or [**GitHub Issues**](https://github.com/ZupIT/docs-charles/issues).
```
Comment (mdcpmoreira) [inline README.md:76]:
```suggestion
Do you have any questions or ideas? Let's chat in our [**Zup Open Source Forum**](https://forum.zup.com.br) or [**GitHub Issues**](https://github.com/ZupIT/docs-charles/issues).
```

**Review by mdcpmoreira — APPROVED** [review]

**Review by barbararochazup — APPROVED** [review]

**Review by mdcpmoreira — APPROVED** [review]

**Review by otaviojava — APPROVED** [review]

**Pull request merged**